dev/benchmarks/macrobenchmarks/lib/src/cubic_bezier.dart

```diff
@@ -0,0 +1,376 @@
+import 'dart:math';
```

Comment:
missing license

please consider writing an automated test that verifies that all source files have a license header

Reply:
Done. Will add a test in another PR.